pymc3/distributions/continuous.py

```diff
@@ -113,6 +129,9 @@ def __init__(self, *args, **kwargs):
         super(Flat, self).__init__(*args, **kwargs)
         self.median = 0
 
+    def random(self, point=None, size=None):
+        return np.zeros(size)
```

Comment:
An uninformative distribution should not return zeros as random draws -- should be a random number on the real line (at least in principle).